### PR TITLE
Combine price and preferred node in price expander

### PR DESCRIPTION
**Pull request by mwielgus**
Ref: #82 

The tests will be rewritten to be specified in a table-like form in the next PR.

### CODE REVIEW

File: cluster-autoscaler/expander/price/price.go [hunk 1]
```diff
@@ -72,13 +114,59 @@ nextoption:
...
 		}
 
-		optionScore := totalNodePrice / totalPodPrice
-		glog.V(5).Infof("Price of %s expansion is %f - ratio %f", option.NodeGroup.Id(), totalNodePrice)
+		// How well the money is spent.
+		priceSubScore := (totalNodePrice + stabilizationPrice) / (totalPodPrice + stabilizationPrice)
+		// How well the node matches generic cluster needs
+		nodeUnfitness := p.nodeUnfitness(preferredNode, nodeInfo.Node())
+
+		// TODO: normalize node count against preferred node.
```
Comment (MaciekPytel) [inline cluster-autoscaler/expander/price/price.go]: Any reason not to do that now? It seems simple enough. Or is it large enough to make it more readable by putting it in separate pr?

Comment (mwielgus) [inline cluster-autoscaler/expander/price/price.go]: The second. I need to think more how to do it properly.

**Comment by MaciekPytel** [comment]
Please fix golint, otherwise lgtm

**Pull request merged**